### ISSUE TITLE
Realease mode Bar chart crashing android

### ISSUE THREAD

**Issue by vickyramz** [indiespirit/react-native-chart-kit#178]
Debug mode Barchart working fine,But in release mode it is crashing

**Comment by nehaj2995** [comment]
Yes, I'm also facing the same issue. Everything works fine in debug mode build ,but when I run release mode build ,the android app crashes.

**Comment by nehaj2995** [comment]
Adding

> -keep public class com.horcrux.svg.** {*;}

to proguard-rules.pro file worked for me

reference: [https://github.com/indiespirit/react-native-chart-kit/issues/115](url)
